test-suite/tests/testpostw90_fe_ahc_adaptandfermi/Fe.wpout

```diff
@@ -0,0 +1,315 @@
+
```

Comment:
As far as I understand, this file, as well as `test-suite/tests/testpostw90_fe_ahc_adaptandfermi/Fe.mmn` should not be included. Especially, `Fe.mmn` is a large file that is intended to exist only locally.
(It seems they are included by the `git add .` command, due to a missing `.gitignore` in this folder: see #352.)

Just deleting them in another commit may *not* be sufficient as these file will remain in the history, but I am not sure.